### ISSUE TITLE
Dormant Usernames

### ISSUE THREAD

**Issue by danpriv** [npm/policies#64]
Something like this would be great!
https://help.github.com/articles/name-squatting-policy/

**Comment by ilg-ul** [comment]
yes, but perhaps it would be good to have a definition of "inactive", even a loose one.

**Comment by danpriv** [comment]
For like 6 months without any activity.

**Comment by ilg-ul** [comment]
then we need to define "activity". publishing a package would be one. responding to emails sent to the address in the profile in less than 30 days might be another one.

and what do you suggest to happen to inactive users? to have the account removed? 
what if the account owns packages that are still downloaded? who will be the owner then?

anyway, the procedure to reclaim a username can be inspired from the one to reclaim a package name (just did this recently), although they are not identical.

**Comment by danpriv** [comment]
If user has no packages and he hasn't logged into his account in the past 6 months then those usernames should be recycled or released upon request (github does the same).

**Comment by ilg-ul** [comment]
... "those usernames **may/might** be recycled upon request" ...

**Comment by gaboesquivel** [comment]
yes, I'd love to see Github name squatting  policy replicated in npm

**Comment by gaboesquivel** [comment]
For anyone landing here, you can email support@npmjs.com and they will address your username request https://www.npmjs.com/policies/disputes
